include/mender-utils.h

```diff
@@ -34,6 +34,12 @@ extern "C" {
 #include <assert.h>
 #include <cJSON.h>
 
+/**
+ * @brief Macro for comparing two strings
+ * @return true if the strings are equal, false otherwise
+ */
+#define StringEqual(str1, str2) (0 == strcmp(str1, str2))
```

Comment:
We might need to extend this to gracefully handle `NULL` values in the future, but good enough for now. :+1:

Reply:
Maybe that should be another macro.  E.g.:
```C
#define SafeStringEqual(str1, str2) ...
```
to avoid adding more code in places where it is not needed?

Reply:
Then I vote for `StringEqual()` and `UnsafeStringEqual()`. :wink: